engines/e_devcrypto.c

```diff
@@ -406,11 +423,15 @@ static int devcrypto_test_cipher(size_t cipher_data_index)
 static void prepare_cipher_methods(void)
 {
     size_t i;
-    struct session_op sess;
+    session_op_t sess;
     unsigned long cipher_mode;
 #ifdef CIOCGSESSINFO
     struct session_info_op siop;
 #endif
+#ifdef CIOCGSESSION2
+    struct crypt_find_op fop;
+    enum devcrypto_accelerated_t accelerated;
+#endif
```

Comment:
I think we will get unused-variable warnings relating to `siop` with the separate `#ifdef`s -- shouldn't we do `#ifdef CIOCGSESSION2`/`#elif defined(CIOCGSESSION)`/`#endif` to match the actual usage of `siop` below?

Reply:
This does assume that no system ever has both.  I think having the clauses match is fine though.